app/controllers/insured/fdsh_ridp_verifications_controller.rb

```diff
@@ -100,7 +100,7 @@ def received_response(event_kind)
     end
 
     def find_response(event_kind)
-      ::Fdsh::Ridp::EligibilityResponseModel.where(event_kind: event_kind, primary_member_hbx_id: @person.primary_family.primary_applicant.hbx_id).to_a.max_by(&:deleted_at)
+      ::Fdsh::Ridp::EligibilityResponseModel.where(event_kind: event_kind, primary_member_hbx_id: @person.primary_family.primary_applicant.hbx_id, :deleted_at.nin => [nil]).to_a.max_by(&:deleted_at)
```

Comment:
- can you update the query to use `.ne` instead of `.nin`?
- Line is too long, can you reduce the length of the line by assigning this `@person.primary_family.primary_applicant.hbx_id` to a vairable?